examples/modals/uploader.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional
+import discord
+from discord import app_commands
+
+import traceback
+
+# The guild in which this slash command will be registered.
+# It is recommended to have a test guild to separate from your "production" bot
```

Comment:
> guild separate from your production "bot"
A guild separated from a bot makes little sense, or at least this reads poorly to me.
Maybe change it to something akin to "test guild before syncing globally" or similar.